### PR TITLE
Updating Filtration Summary KDE plots

### PR DESCRIPTION
**Pull request by gwaygenomics**
The only change here is renaming the samples. All files and figures reflect this change.

### CODE REVIEW

File: scripts/nbconverted/3.variant_allele_frequency.py [hunk 1]
```diff
@@ -21,31 +22,46 @@
...
+    
+    pheno_subset_df = pheno_df.query('final_id == @final_id')
     
     # generate filenames for each unique sample
     fig_name = os.path.join('figures', 'sift_gnomad', 'replicates',
-                            '{}_sift_gnomad_kde.pdf'.format(unique_sample))
+                            '{}_sift_gnomad_kde.pdf'.format(final_id))
     multi_plots = []
-    for wes_id in phenotype_subset_df['wes_id']:
+    for wes_id in set(pheno_subset_df['wes_id']):
```
Comment (danich1) [inline scripts/nbconverted/3.variant_allele_frequency.py]: Is there a reason for a set here?

Comment (gwaygenomics) [reply scripts/nbconverted/3.variant_allele_frequency.py]: yeah - the column `wes_id` is redundant in the dataframe. I only want to loop through one instance of the id

Comment (danich1) [reply scripts/nbconverted/3.variant_allele_frequency.py]: Ah ok. Well I'll approve once @huqiwen0313 has taken a look through.

Comment (gwaygenomics) [reply scripts/nbconverted/3.variant_allele_frequency.py]: in this case, I think it is ok to merge with only one reviewer (it is also on the onboarding). I am looking to get these updates out before I leave for SF

Comment (danich1) [reply scripts/nbconverted/3.variant_allele_frequency.py]: ah ok approved in this case.

**Pull request merged**